Source/Core/DolphinQt/Config/Mapping/WiimoteEmuMetroid.cpp

```diff
@@ -95,6 +161,14 @@ void WiimoteEmuMetroid::CreateMainLayout()
       GetPort(), WiimoteEmu::WiimoteGroup::Visors));
   groupbox1->addWidget(visor_box);
 
+  auto* morphball_control_box = CreateGroupBox(tr("Morphball Controller Profile"), Wiimote::GetWiimoteGroup(
+    GetPort(), WiimoteEmu::WiimoteGroup::MorphballControls));
+  groupbox1->addWidget(morphball_control_box);
+  m_morphball_combobox = (morphball_control_box->findChild<QComboBox*>(tr("ProfileList")));
+  m_morphball_combobox->setToolTip(tr("Set the controller profile to use\nwhen in Morph Ball in MP3."));
+
```

Comment:
The combo box shouldn't let you write/edit text in it. It's unlikely but aside from being potentially confusing/unclean, some user may try write their profile name into the box and not understand why it doesn't work.